release_notes/ocp-4-6-release-notes.adoc

```diff
@@ -433,6 +433,30 @@ from both sides of the modal are available for display at one time. Clicking
 *Save* will save the changes that you make. Clicking *Restore Default Columns*
 will restore the default settings of the columns.
 
+[id="ocp-4-6-web-console-administrator-perspective"]
+==== Administrator Perspective
+
+* The *Pipelines* view was updated to provide details on triggers, event listeners, trigger templates, trigger bindings, and cluster trigger bindings.
```

Comment:
The *Pipelines* navigation item was updated to provide tab views for Pipelines, Tasks and Triggers. Triggers will include tabs for Event Listeners, Trigger Templates, Trigger Bindings, and Cluster Trigger Bindings.